src/corehost/cli/dotnet/dotnet.manifest

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
+  <compatibility xmlns="urn:schemas-microsoft-com:compatibility.v1">
+    <application>
```

Comment:
You should look at the default manifest provided by the tool chain (and already in dotnet.exe) and merge it with this.  I believe it has things like requested elevation level.

Reply:
That is already done by the linker - it merges all manifests - so this one and the one generate by it with the existing settings.

Reply:
Cool, so long as you've verified that merge behavior that's good.